src/main/java/org/jenkinsci/plugins/matrixauth/AuthorizationMatrixNodeProperty.java

```diff
@@ -187,7 +187,7 @@ public AuthorizationMatrixNodeProperty newInstance(StaplerRequest req, @Nonnull
 
         @Override
         public boolean isApplicable(Class<? extends Node> node) {
-            return Node.class.isAssignableFrom(node) && isApplicable();
+            return isApplicable();
```

Comment:
This only existed as a hack to prevent Kubernetes Plugin from showing the node property, as it passes a wrong class argument.